pages/linux/sattach.md

```diff
@@ -0,0 +1,16 @@
+# sattach
+
+> Attach to a Slurm job step.
+> More information: <https://slurm.schedmd.com/sattach.html>.
+
+- Make available the IO streams (`stdout`, `stderr`, and `stdin`) of a Slurm job step:
+
+`sattach {{jobid}}.{{stepid}}`
+
+- Transmit `stdin` to the specified task only:
+
+`sattach --input-filter {{task_number}}`
+
+- Only display `stdin`/`stderr` of the specified task:
```

Comment:
Please try to keep the language consistent. In this page alone you've used `display`, `transmit`, and `make available`. This is very confusing.

Reply:
```suggestion
- Only redirect `stdin`/`stderr` of the specified task:
```